src/components/pipelines-tasks/TasksRow.tsx

```diff
@@ -73,7 +75,7 @@ const TaskRow: React.FC<RowProps<K8sResourceCommon>> = ({
       <TableData activeColumnIDs={activeColumnIDs} id="name">
         <ResourceLink
           kind={getReferenceForModel(TaskModel)}
-          name={obj.metadata.name}
+          name={getTaskName(obj)}
```

Comment:
It will break the details page link. Should make use of `displayName` props
```suggestion
          name={obj.metadata.name}
          displayName={getTaskName(obj)}
```

IMO we should also update the name on the details page. It might confuse the user to see the difference in name in list page and details page.

Reply:
https://github.com/user-attachments/assets/d9aed72b-3107-4d77-a014-38dd288a7e4a